src/Adapters/Auth/OAuth1Client.js

```diff
@@ -1,7 +1,11 @@
 var https = require('https'),
   crypto = require('crypto');
+var Parse = require('parse/node').Parse;
 
 var OAuth = function(options) {
+  if(!options) {
+    throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND, 'No options passed to OAuth');
```

Comment:
Yea that makes sense, considering case there's no way to fire the exception unless you call it without the proper object.